managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -3066,21 +3068,25 @@ boolean shouldCloseLedger(LedgerHandle lh) {
         }
     }
 
-    void switchToNewLedger(final LedgerHandle lh, final VoidCallback callback) {
+    void switchToNewLedger(@Nonnull LedgerHandle lh, VoidCallback callback) {
```

Comment:
I didn't get why should we add a `@Nonnull` annotation to `lh`. With this annotation, it seems that `lh` should not be null while `callback` might be `null`. However, `callback` should also be non-null.

Reply:
I think we can close both.